Header.c

```diff
@@ -86,10 +86,9 @@ static void Header_addMeterByName(Header* this, const char* name, MeterModeId mo
    unsigned int param = 0;
    size_t nameLen;
    if (paren) {
-      int ok = sscanf(paren, "(%10u)", &param); // CPUMeter
-      if (!ok) {
+      if (sscanf(paren, "(%10u)", &param) != 1) { // CPUMeter
```

Comment:
Shouldn't this be `==` here instead?

Reply:
No, we want to enter the block and scan for a dynamic Meter if the parameter does not match a number.